### DISCUSSION TITLE
Trying to query historical averages (esp for frost dates)

### DISCUSSION THREAD

**Discussion by brianwdavis** [weather-gov/api#213] [Q&A]
I have a web application that is currently using 5 years of local weather station data to calculate frost date averages for users, but this is a limited dataset due to the number of stations and number of years. Is there an endpoint to get the (for example) 30-year averages for a grid point? Particularly interested in frost dates but monthly average temperatures, etc would be useful as well.

**Comment by JJKraw** [comment]
As far as I know, the API isn't the place to go looking for historical data.

**Comment by brianwdavis** [comment]
Thanks! I'll keep looking elsewhere.

**Comment by StephenClouse** [comment]
Historical data can be requested via [NCEI](https://www.ncei.noaa.gov/access). They even have their own API for it :)

Due to various reasons we are only able to offer a limited set of recent data via our API.
